github-actions/slash-commands/main.ts

```diff
@@ -0,0 +1,90 @@
+import * as core from '@actions/core';
+import {context} from '@actions/github';
+import {rerunCircleCi} from './commands/rerun-circleci';
+import {rebase} from './commands/rebase';
+
+/** The marker used in comments to note a command is being made in the comments. */
+const commandMarker = '/ng-bot';
+/** The full command provided in the comment, split on spaces from the comment. */
+let parsedCommand: [string, ...string[]];
+
+/** Get the parsed command from the comment. */
+function getParsedCommand() {
+  if (parsedCommand) {
+    return parsedCommand;
+  }
+  const comment = context.payload.comment!.body || '';
+  const command = (comment.split('\n')[0] || '')
+    .split(' ')
+    .filter((word: string) => !!word)
+    .map((word: string) => word.trim());
+
+  if (command[0] !== commandMarker) {
+    throw Error();
+  }
+  parsedCommand = command.slice(1);
+
+  return parsedCommand;
```

Comment:
Why not use a multi-line regex like `^/ng-bot (.*)$` for all of this? Also any strong reason to cache the command here? I think it would be cleaner to just pass it around.